Resources/templates/Doctrine/ListBuilderAction.php.twig

```diff
@@ -112,9 +111,21 @@
 {% endblock %}
 
 {% block addJoinFor -%}
-    protected function addJoinFor($table, $query)
+    protected function addJoinFor($table, $query, $currentObject = true)
     {
-        $query->leftJoin('q.'.$table, $table);
+        if($currentObject){
+            $query->leftJoin('q.'.$table, $table);
+        } else {
+            $tables = explode('.', $table);
+            $previousTable = 'q';
+            foreach(array_slice($tables, 0, -1) as $table)
```

Comment:
@tobias-93 according to PSR-2 only class, interface and methods left brace starts in newline,
anonymous functions and loops (foreach, while, for) go inline

should be:
```php
foreach(array_slice($tables, 0, -1) as $table) {
```